docs/ref/csrf.txt

```diff
@@ -462,3 +472,34 @@ A number of settings can be used to control Django's CSRF behavior:
 * :setting:`CSRF_FAILURE_VIEW`
 * :setting:`CSRF_HEADER_NAME`
 * :setting:`CSRF_TRUSTED_ORIGINS`
+
+Frequently Asked Questions
+==========================
+
+Is posting an arbitrary CSRF token pair (cookie and POST data) a vulnerability?
+-------------------------------------------------------------------------------
```

Comment:
While this is true of the current CSRF protection, PR #5605 makes the token much less arbitrary. We are not certain if that PR will be merged, and either way the question should be answered for older Django's, but I thought a heads-up is warranted.